src/index.html

```diff
@@ -7,6 +7,355 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="./images/logo.png" alt="logo_here">
```

Comment:
change alt text to something clearer (like "MOYO logo")

Reply:
+ fixed it